tests/dat/actions/echoForSequences.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Returns params, or an empty string if no parameter values are provided
+ */
+function main(params) {
+    return params || {};
+}
```

Comment:
In some tests, a Sequence with some actionnames is created.
This name is used for the actionname and the filename.
I could change this, if you want, but I think it is more clear, if this test structure is kept.